### PR TITLE
integration specs (refs #38321)

### PR DESCRIPTION
**Pull request by born4new**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM docker-registry.nine.ch/ninech/ruby:trusty
```
Comment (calmyournerves) [inline Dockerfile:1]: Could we switch this to an official image?

**Review by calmyournerves — APPROVED** [review]
Nice first step! Can we switch this to Travis as well (not sure if they support Docker)?

File: Dockerfile (hunk 1 above)
Comment (born4new) [reply Dockerfile:1]: Yes. I'll make two PRs: one for Docker, the other one for the integration specs.

**Comment by born4new** [comment]
Closed for #6 and #7

**Pull request closed without merging**